app/controllers/application_controller.rb

```diff
@@ -1,15 +1,18 @@
 class ApplicationController < ActionController::Base
-
+  #, if: :storable_location?
```

Comment:
@kyriakipapadimitriou @julies642 Can you explain what this line does? Seems like a single line comment(#), and if it is, a descriptive comment would be better.

Reply:
yes it was a leftover commented out code from an implementation I tried initially. This line has been now deleted. Thanks for the catch!